backend/src/services/file.service.ts

```diff
@@ -1,18 +1,37 @@
 import fs from 'fs';
 import { log } from '../helper/logger.js';
 import { Request, Response } from 'express';
+import { mkdir, writeFile } from 'node:fs/promises';
 
-export async function sendFile(req: Request, res: Response, path: string, contentType: string): Promise<void> {
-  fs.readFile(path, function (err, data) {
-    if (err) {
-      log.error('Requested file was not found.');
-      res.writeHead(404);
-      res.end(JSON.stringify(err));
-      return;
-    }
-    log.info(`The file "${path}" was sent to the frontend.` );
-    res.setHeader('Content-Type', contentType);
-    res.writeHead(200);
-    res.end(data);
-  });
+export class FileService {
+  async sendFile(req: Request, res: Response, path: string, contentType: string): Promise<void> {
+    fs.readFile(path, function (err, data) {
+      if (err) {
+        log.error('Requested file was not found.');
+        res.writeHead(404);
+        res.end(JSON.stringify(err));
+        return;
+      }
+      log.info(`The file "${path}" was sent to the frontend.`);
+      res.setHeader('Content-Type', contentType);
+      res.writeHead(200);
+      res.end(data);
+    });
+  }
+
+  fileExists(filePath: string): boolean {
+    return fs.existsSync(filePath);
+  }
+
+  directoryExists(directory: string) {
+    return fs.existsSync(directory);
+  }
+
+  async createLogFile(file: string, content: string) {
```

Comment:
This service is separated from logger, so I think you can name it just 'createFile'